docs/config.md

```diff
@@ -662,6 +662,33 @@ then the `shape` option of the `perf_analyzer_flags` option must be specified.
 More information about this can be found in the 
 [Perf Analyzer documentation](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md#input-data).
 
+**Note**:
+Perf Analyzer now supports SSL via GRPC and HTTP. It can be enabled via Model Analyzer configuration file updates.
+
+GRPC example:
+
+```yaml
+model_repository: /path/to/model/repository/
+profile_models:
+  model_1:
+    perf_analyzer_flags:
+        ssl-grpc-root-certifications-file: /path/to/PEM/encoded/server/root/cert
+```
+
+HTTPS example:
+
+```yaml
+model_repository: /path/to/model/repository/
+profile_models:
+  model_1:
+    perf_analyzer_flags:
+        ssl-https-ca-certificates-file: /path/to/cert/authority/cert/file
+        ssl-https-client-certificate-file: /path/to/client/cert/file
+        ssl-https-private-key-file: /path/to/private/key/file
+```
+More information about this can be found in the 
+[Perf Analyzer documentation](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md).
```

Comment:
can you link to this directly:

https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md#ssltls-support

Reply:
done